### ISSUE TITLE
Sidebar collapse arrow give accessibility error for no alt text or aria-label

### ISSUE THREAD

**Issue by natac13** [facebook/docusaurus#4010]
## 🐛 Bug Report

I did a quick check of my site and noticed that the Collapse sidebar [`<IconArrow />`](https://github.com/facebook/docusaurus/blob/397fde0b2f1b4005facb42d3a637725fd47295a8/packages/docusaurus-theme-classic/src/theme/IconArrow/index.tsx) give a error about no alt text. Current on `v.Alpha-70` the edit page button had the same issue but I see it was [fixed](https://github.com/facebook/docusaurus/blob/c4cbe8ec724ef47b47d1460a7e60ab1fea04b664/packages/docusaurus-theme-classic/src/theme/IconEdit/index.tsx#L25) and is now present in `71`

Ill submit a PR for this and link it here.
I was thinking of just passing the `aria-label` as a prop to `IconArrow` as it is used in 2 spots: [collapse](https://github.com/facebook/docusaurus/blob/397fde0b2f1b4005facb42d3a637725fd47295a8/packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx#L271-L281) and [expand the side bar](https://github.com/facebook/docusaurus/blob/397fde0b2f1b4005facb42d3a637725fd47295a8/packages/docusaurus-theme-classic/src/theme/DocPage/index.tsx#L94-L103).
Thought this would be easiest As [props](https://github.com/facebook/docusaurus/blob/c4cbe8ec724ef47b47d1460a7e60ab1fea04b664/packages/docusaurus-theme-classic/src/theme/IconArrow/index.tsx#L13) are being spread out on the `svg`

## To Reproduce

1. Go to Docusaurus doc site
2. run [Accessibility insights for web](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)

## Expected behavior

No errors

![docusaurus-sidebar-collaspe-aria-label](https://user-images.githubusercontent.com/10660283/103916836-e6c9a780-50da-11eb-9fe1-49502a754e9b.png)
